src/api/app.js

```diff
@@ -49,6 +50,10 @@ app.use(express.json());
 app.use(express.urlencoded({extended: false}));
 app.use(compression());
 
+const redisClient = createClient({
```

Comment:
Due to `connect-redis` no longer bundling `redis`, we now have to create the client ourselves and pass it in, here and in the server app.